enhancement: Add filter options to the list policies method of the admin API (#313)
cerbos/cerbos: docs/modules/cli/pages/ctl.adoc

----
cerbosctl decisions --tail=20
----

== `list`

> This command lists the policies stored in the Cerbos server. It accepts the filters below to enable filtering the list. The filters use JSONPath expressions to query fields.

Reviewer: ```suggestion
This command lists the policies available in the configured policy repository. The list can be filtered by specifying one or more pairs of field selectors and match values as command line arguments. Field selectors are expected to be valid JSONPath expressions describing policy fields.
```

> See link:https://tools.ietf.org/id/draft-goessner-dispatch-jsonpath-00.html#table-1[specification] for detailed information about JSONPath.

Reviewer: ```suggestion
See link:https://tools.ietf.org/id/draft-goessner-dispatch-jsonpath-00.html#table-1[specification] for more information about JSONPath.
```

> [#list-filters]
> .Filters
> ****
> 
> --field-eq:: Filter a field with an exact match
> + 
> - `--field-eq="$.resourcePolicy.version=20210210"`: Will list resource policies with version 20210210.
> - `--field-eq="$.*.version=20210210"`: Will list any type of policies with version 20210210.
> --field-match:: Filter a field with a regex match. 
> +
> - `--field-match="$.resourcePolicy.resource=leave"`: Will list the resource policies with "leave" contained in the resource field.
> - `--field-match="$.resourcePolicy.version=\d"`: Will list resource policies with a version includes a numeric value.
> 
> ****

Reviewer: Let's keep the example style consistent with the rest of the documentation.
 
```suggestion
Use `--field-eq` to perform an exact match and `--field-match` to perform a regular expression match.

.List resource policies where the `version` field is exactly `20210210`
----
cerbosctl list --field-eq='$.resourcePolicy.version=20210210'
----

.List any policy where `version` is exactly `20210210`
----
cerbosctl list --field-eq='$.*.version=20210210'
----

.List resource policies where the `resource` field contains `leave`
----
cerbosctl list --field-match='$.resourcePolicy.resource=leave'
----

.List resource policies with a numeric version
----
cerbosctl list --field-match='$.resourcePolicy.version=\d+'
----
```

> 
> == `version`
> 
> This command displays the detailed version information of the cerbosctl and the Cerbos server it connected.

Reviewer: I don't think we need this section here.

```suggestion

```
